cuda/private/cuda_helper.bzl

```diff
@@ -250,6 +255,55 @@ def _get_cc_sysroot(ctx):
     cuda_toolchain = find_cuda_toolchain(ctx)
     return getattr(cuda_toolchain, "cc_sysroot", None)
 
+def _get_cc_host_compile_flags(ctx):
```

Comment:
I believe you want to move this to toolchain.bzl, together with `_get_cc_sysroot`, otherwise, you will need to query those flags on every cuda related actions!

Reply:
I think we do need to do that, since the flags are dependent on the features enabled for the particular rule